api/v1alpha1/nicclusterpolicy_types.go

```diff
@@ -55,6 +55,18 @@ type PodProbeSpec struct {
 	PeriodSeconds       int `json:"periodSeconds"`
 }
 
+// DriverCertConfigSpec defines custom certificates configuration for driver container
+type DriverCertConfigSpec struct {
+	// +kubebuilder:validation:Optional
+	Name string `json:"name,omitempty"`
```

Comment:
nit: if fields has "omitempty" json tag it will be optional by default

Reply:
do you see an issue in being explicit here with the kubebuilder annotations ?

Reply:
No technical reasons. For the network-operator we use various technics to mark a field as optional. As far as I can see, this style is not used anywhere in the project yet. I prefer to keep things consistent, but I don't have a preference for a specific style.
This is a "nit" comment and can be completely ignored.